plugins/catalog-backend/config.d.ts

```diff
@@ -38,6 +38,33 @@ export interface Config {
        * E.g. ["Component", "API", "Template", "Location"]
        */
       allow: Array<string>;
+      /**
+       * Limit this rule to a specific location
+       *
+       * Example with a fixed location
+       *  { "type": "url", "exact": "https://github.com/a/b/blob/file.yaml"}
+       *
+       * Example using a Regex
```

Comment:
Not a regex anymore, it's a glob pattern

Reply:
Yes. That is true. I'll change the wording